draft-irtf-nwcrg-coding-and-congestion-04.xml

```diff
@@ -429,25 +429,25 @@
         <section anchor="sec:research" title="Open research questions">
 
 		<t>This section provides a simplified state-of-the art of the activities related to congestion control and coding. The objective is to identify open research questions and contribute to advice when evaluating coding mechanisms.</t>
-			<section title="Activity related to congestion control and coding">
-				<t>The identification of activities related to congestion control and coding should be mapped with the organization presented in this document:<list style="symbols"> 
+			<section title="Activities related to congestion control and coding">
+				<t>We map activities related to congestion control and coding with the organization presented in this document:<list style="symbols">
 						<t>For the FEC above transport case: TBD</t>	
 						<t>For the FEC within transport case: <xref target="I-D.swett-nwcrg-coding-for-quic"></xref>, <xref target="QUIC-FEC"></xref>, <xref target="RFC5109"></xref>.</t>	
 						<t>For the FEC below transport case: <xref target="NCTCP"></xref>, <xref target="I-D.detchart-nwcrg-tetrys"></xref>.</t>	
 				</list></t>
 			</section>
 
 			<section title="Open research questions">
-				<t>The research questions should be mapped following the oranization of this document. In all these three use-cases, a open questions remains and is inherent to the usage of coding. There is a trade-off between reducing goodput, helping to recover from /
-					and congestion losses, being fair in respect with non-coded flows and avoiding transmitting useless repaired symbols.</t>
-				<t>For the FEC above transport case, there is a trade-off between the amount of redundancy to add, as a function of the transport layer protocol and application requirements.</t>
-				<t>For the FEC within transport case, the addition of coding may hide congestion losses and proposing methods to signal such events to end point could be propose.</t>
-				<t>For the FEC below transport case, there are opportunities for introducing interaction between congestion control and coding schemes to improve the quality of experience while guarantee fairness with other flows. An open question also resides on the relevance of the FEC when there are multiple streams that exploit the FEC channel.</t>
+				<t>The research questions should be mapped following the organization of this document. In all these three use-cases, open questions remain. There is a general trade-off, inherent to the use of coding, between reducing goodput (helping to recover from /
```

Comment:
```suggestion
				<t>The research questions should be mapped following the organization of this document. In all these three use-cases, open questions remain. There is a general trade-off, inherent to the use of coding, between (1) reducing goodput when useless repaired symbols are transmitted and (2) helping to recover from transmission and congestion losses.</t>
```

Reply:
What do you think?

Reply:
Looks great to me !

Reply:
Just:
repaired symbols -> repair symbols 

:)